docker-compose.yml

```diff
@@ -156,18 +156,41 @@ services:
         soft: 262144
         hard: 262144
     environment:
-      - LOG_FORMAT=${LOG_FORMAT}
-      - HTTP_PORT=${HTTP_PORT}
+      - LOG_LEVEL=${LOG_LEVEL:-info}
+      - LOG_FORMAT=${LOG_FORMAT:-json}
+      - HTTP_PORT=${HTTP_PORT:-8080}
       - DB_HOST=${DB_HOST}
-      - DB_PORT=${DB_PORT}
+      - DB_PORT=${DB_PORT:-8123}
       - DB_USER=${DB_USER}
       - DB_PASSWORD=${DB_PASSWORD}
       - DB_NAME=${DB_NAME}
-      - START_EPOCH=${START_EPOCH}
+      - DB_MAX_RETRIES=${DB_MAX_RETRIES:-10}
+      - DB_MIN_BACKOFF_SEC=${DB_MIN_BACKOFF_SEC:-1}
+      - DB_MAX_BACKOFF_SEC=${DB_MAX_BACKOFF_SEC:-120}
+      - START_EPOCH=${START_EPOCH:-155000}
       - ETH_NETWORK=${ETH_NETWORK}
-      - VALIDATOR_REGISTRY_SOURCE=${VALIDATOR_REGISTRY_SOURCE}
+      - VALIDATOR_REGISTRY_SOURCE=${VALIDATOR_REGISTRY_SOURCE:-lido}
+      - VALIDATOR_REGISTRY_FILE_SOURCE_PATH=${VALIDATOR_REGISTRY_FILE_SOURCE_PATH:-./docker/validators/custom_mainnet.yaml}
+      - VALIDATOR_REGISTRY_LIDO_SOURCE_SQLITE_CACHE_PATH=${VALIDATOR_REGISTRY_LIDO_SOURCE_SQLITE_CACHE_PATH:-./docker/validators/lido_mainnet.db}
+      - VALIDATOR_USE_STUCK_KEYS_FILE=${VALIDATOR_USE_STUCK_KEYS_FILE:-false}
+      - VALIDATOR_STUCK_KEYS_FILE_PATH=${VALIDATOR_STUCK_KEYS_FILE_PATH:-./docker/validators/stuck_keys.yaml}
       - EL_RPC_URLS=${EL_RPC_URLS}
       - CL_API_URLS=${CL_API_URLS}
+      - CL_API_RETRY_DELAY_MS=${CL_API_RETRY_DELAY_MS:-500}
+      - CL_API_GET_RESPONSE_TIMEOUT=${CL_API_GET_RESPONSE_TIMEOUT:-15000}
+      - CL_API_MAX_RETRIES=${CL_API_MAX_RETRIES:-1}
+      - CL_API_GET_BLOCK_INFO_MAX_RETRIES=${CL_API_GET_BLOCK_INFO_MAX_RETRIES:-1}
+      - FETCH_INTERVAL_SLOTS=${FETCH_INTERVAL_SLOTS:-32}
+      - CHAIN_SLOT_TIME_SECONDS=${CHAIN_SLOT_TIME_SECONDS:-12}
+      - WORKING_MODE=${WORKING_MODE:-finalized}
+      - NODE_ENV=${NODE_ENV:-development}
+      - DRY_RUN=${DRY_RUN:-false}
```

Comment:
let's move them up. very important vars

Reply:
Agree. Moved these vars to the beginning of the `environment` section.